packages/lesswrong/server/recombee/client.ts

```diff
@@ -353,6 +353,10 @@ const helpers = {
       : context.ReadStatuses.find({ postId: { $in: curatedPostIds.slice(1) }, userId, isRead: true }).fetch();
   },
 
+  getManuallyStickiedPostsReadStatuses(manuallyStickiedPostIds: string[], userId: string, context: ResolverContext) {
```

Comment:
I don't think we want to pass in a list of `manuallyStickiedPostIds`, since we can just get the post id from the setting and the caller doesn't already have it.